### PR TITLE
Index allowExtraTime in RAM minter configuration extraMinterDetails

### PR DESCRIPTION
**Pull request by yoshiwarab**
## Description of the change

Index allowExtraTime in RAM minter configuration extraMinterDetails. Index `configuredAuctionEndTime` on RAM minter configuration extraMinterDetails to differentiate between the end time that may have been extended due to a last minute bid.

https://linear.app/art-blocks/issue/PLT-861/index-allowextratime-in-subgraph

>reminder: Any subgraph deployments should be documented as [Releases](https://github.com/ArtBlocks/artblocks-subgraph/releases) in this repository.

>reminder: Any changes to subgraph schema should be accompanied by corresponding changes to the Art Blocks documentation site, available at https://docs.artblocks.io/ (specifically, see the [Subgraph Entities](https://docs.artblocks.io/creator-docs/art-blocks-api/entities/) and [Subgraph Querying and API Overview](https://docs.artblocks.io/creator-docs/art-blocks-api/queries/) sections).

### CODE REVIEW

**Review by lindsgil — APPROVED** [review]
🔥 ⌛

File: tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts [hunk 1]
```diff
@@ -510,41 +543,91 @@ describe("RAMLib event handling", () => {
...
+      // The configuredAuctionEndTime is only updated when receiving an AuctionConfigUpdated event,
+      // not when receiving AuctionTimestampEndUpdated. While AuctionTimestampEndUpdated is emitted
+      // in three cases:
+      // 1. When a bid in the last 5 minutes extends the auction (most common)
+      // 2. When reduceAuctionLength is called
+      // 3. When adminAddEmergencyAuctionHours is called
+      // Ideally, we would update configuredAuctionEndTime for cases 2 and 3. However, since
+      // these are edge cases and we cannot reliably differentiate them from the more common
+      // case 1, we err on the side of not updating configuredAuctionEndTime in the
+      // AuctionTimestampEndUpdated handler.
```
Comment (ryley-o) [inline tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts:574]: recommend a similar comment in `ram-lib-mapping::handleAuctionTimestampEndUpdated`

**Review by ryley-o — APPROVED** [review]
LGTM!

File: tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts (hunk 1 above)
Comment (yoshiwarab) [reply tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts:574]: Done!

**Pull request merged**